src/index.html

```diff
@@ -4,10 +4,309 @@
+      <h2 class="heading section__heading">Testimonials<span class="heading__crown"></span></h2>
+      <div class="testimonials">
+        <div class="person testimonials__person">
+          <img
+            class="person__photo"
+            src="./images/people/1.png"
+            alt="person photo"
+          >
+          <h3 class="person__name">Leslie Mckinney</h3>
+          <p class="person__position">CEO</p>
+          <p class="person__content">Nice work, Certificates National Association of Realtors (USA)</p>
+        </div>
+        <div class="person testimonials__person">
+          <img
+            class="person__photo"
+            src="./images/people/2.png"
+            alt="person photo"
+          >
+          <h3 class="person__name">Scarlett Hawkins</h3>
+          <p class="person__position">Senior Vice President</p>
+          <p class="person__content">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart</p>
+        </div>
+        <div class="person testimonials__person">
+          <img
+            class="person__photo"
+            src="./images/people/3.png"
+            alt="person photo"
+          >
+          <h3 class="person__name">Max Henry</h3>
+          <p class="person__position">Chief Design Officer</p>
+          <p class="person__content">When my lovely valley of steam rises and half-day sun is above an impermeable</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="section section--short" id="contact">
+      <h2 class="heading section__heading">Contact us<span class="heading__crown"></span></h2>
+      <div class="contact">
+        <div class="contact__info">
+          <h3 class="contact__label">Phone</h3>
+          <a class="contact__text" href="tel:+1 234 555-55-55">+1 234 555-55-55</a>
+          <h3 class="contact__label">Email</h3>
+          <a class="contact__text" href="mailto:hello@miami.com">hello@miami.com</a>
+        </div>
+        <div class="contact__address">
+          <div class="contact__label">Address</div>
+          <div class="contact__text">400 first ave. n.suite 700<br>Minneapolis, MN 55401</div>
+        </div>
+        <form class="contact__form form"
+            action="#"
+            onsubmit="onSubmit()"
+        >
+          <div class="form__line">
+            <input
+              type="text"
+              class="input"
+              placeholder="Name"
+              required
+            >
+          </div>
+          <div class="form__line">
+            <input
+              type="text"
+              class="input"
+              placeholder="Email"
+              required
+            >
+          </div>
+          <div class="form__line">
+            <textarea
+              name="message"
+              id="message"
+              class="input form__area"
+              placeholder="Message"
+            >
+            </textarea>
```

Comment:
```suggestion
            <textarea
              name="message"
              id="message"
              class="input form__area"
              placeholder="Message"
           ></textarea>
```
ось так має з'явитись placeholder

Reply:
пробувала, але lint не пропускає